ebuild-writing/functions/src_test/text.xml

```diff
@@ -96,12 +95,6 @@ using <c>sed</c> or skipping a particular <c>make</c> target is
 sufficient.
 </p>
 
-<note>
-<c>emake</c> should not be used for <c>src_test</c> <d/> trying to
-parallelise tests unless the <c>Makefile</c> was specifically designed
-for this can cause all sorts of strange problems.
-</note>
-
```

Comment:
I don't think there's a point in making a special verbose notes about past state not applying anymore.